.circleci/config.yml

```diff
@@ -10,7 +10,6 @@ jobs:
      resource_class: xlarge
      working_directory: /source
      environment:
-       BAZEL_REMOTE_CACHE: https://storage.googleapis.com/envoy-circleci-bazel-cache/
```

Comment:
Let's revert these; if we are going to disable the cache, it should be a separate PR.